docs/release-notes.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Release Notes
+---
+
+## v2.0.0
+
+### New Features
+
```

Comment:
- Support for [LoRa Basics™ Station](https://github.com/lorabasics/basicstation/tree/v2.0.5) LNS Protocol (v2.0.5)
- Support for [LoRa Basics™ Station](https://github.com/lorabasics/basicstation/tree/v2.0.5) CUPS Protocol (v2.0.5) for credential management and firmware upgrade
- LoRaWAN Network Server and Basics™ Station can be decoupled, allowing the possibility to run multiple concentrators on a single LoRaWAN Network Server instance
- Support for secure communication between LoRaWAN Network Server and Basics™ Station
- Support for running LoRaWAN Network Server on Azure IoT Edge for Linux on Windows
- Regional support for AS923 and CN470 (Regional Parameters v1.0.3rA and RP002-1.0.3 revisions)
- Observability: Integration with Azure Monitor
- Observability: exposing a Prometheus endpoint for metrics scraping
- Observability: @bastbu please add more here 

I would also add a "Other" or "Code quality" section with:
- Introduction of LoRaWAN Primitives and refactoring of existing code to make wide use of those
- Upgrade to .NET 6 (LTS version)
- Upgrade container base images to Debian 11
- Improvements in E2E CI, continuously testing multiple scenarios and authentication modes
- @p-schuler is there something we want to write here or in the breaking changes regarding the device caching?

Reply:
> LoRaWAN Network Server and Basics™ Station can be decoupled, allowing the possibility to run multiple concentrators on a single LoRaWAN Network Server instance

Correction: allowing the possibility to **connect** multiple concentrators to a single LNS.

Reply:
There are not breaking changes to the device caching. It's been changed and improved, but not breaking compatibility.

Reply:
Observability: default monitoring for edgeAgent and edgeHub modules.

Reply:
Observability: distributed tracing